pyproject.toml

```
@@ -214,6 +214,7 @@ build-backend = "hatchling.build"
       "F821",    # undefined name '...'  <- jaxtyping
       "FIX002",  # Line contains TODO
       "ISC001",  # Conflicts with formatter
+      "N806",    # Variable in function should be lowercase
```

Can we do this more locally? Or it's too much hassle for this PR, then just ... 
```suggestion
      "N806",    # Variable in function should be lowercase  # TODO: address locally
```

---

Yea, I can - this is just a formatting rule I totally disagree with!! Especially for physics-related code, where I often have a `delta` and a `Delta` that mean different things, for example :)

---

Very true! Whenever I have a physics-heavy module I just turn it off for that whole file.

---

OK fixed in the latest batch of commits